photogrammetry/pom.xml

```diff
@@ -86,6 +86,7 @@
 			<plugin>
 				<groupId>org.sonatype.plugins</groupId>
 				<artifactId>nexus-staging-maven-plugin</artifactId>
+				<version>${nexus-staging-maven-plugin.version}</version>
```

Comment:
Good catch.   However, to stay consistent, we usually define the versions in the parent pom.xml.   We should probably add the plugin and the version to the pluginManagement section of the parent pom and leave the version blank here.